libs/dal/src/repositories/notification-template/notification-template.repository.ts

```diff
@@ -234,14 +240,14 @@ export class NotificationTemplateRepository extends BaseRepository<
     return this.mapEntities(items);
   }
 
-  async delete(query: NotificationTemplateQuery) {
+  async delete(query: NotificationTemplateByIdQuery) {
```

Comment:
Nitpicky one: This naming still feels a bit off. I think its previous name without the `id` was better.

Reply:
it's being used internally

Reply:
Still, inner DX matters :)

Reply:
I'll change it